classes/ActionScheduler_Store.php

```diff
@@ -188,13 +188,43 @@ public function get_status_labels() {
 
 	public function init() {}
 
+	/**
+	 * Mark an action that failed to fetch correctly as failed.
+	 *
+	 * @since 2.2.6
+	 *
+	 * @param int $action_id The ID of the action.
+	 */
+	public function mark_failed_fetch_action( $action_id ) {
+		self::$store->mark_failure( $action_id );
+	}
+
+	/**
+	 * Add base hooks
+	 *
+	 * @since 2.2.6
+	 */
+	protected static function hook() {
+		add_action( 'action_scheduler_failed_fetch_action', array( self::$store, 'mark_failed_fetch_action' ), 20 );
```

Comment:
Should this be just `self` rather than `self::$store`? Same question for the `unhook()` method below.

Reply:
I intended for the action to run using the class of the `$store` instance rather than the base class. That allows an inheriting class to override the hook/unhook functions without having to create a second set of handlers.